R/boxr_file_versions.R

```diff
@@ -1,13 +1,18 @@
-#' Get details of previous versions of a Box file
+#' Get details about versions of a Box file
 #' 
-#' Box explicitly versions files; this function returns a
+#' Box uses file versioning, but the API does not explicitly provide version numbers.
+#' These functions use `modified_date` as a proxy and allow programatic version access
+#' in `box_dl()` and `box_read()`
+#' 
+#' * `box_previous_version()` returns a
 #' `data.frame` containing information on a file's previous 
-#' versions on Box. No information about the current version of the file is
-#' returned. If the version of a file is one, then NULL is returned invisibly
+#' versions on Box, the column `file_version_id` can be passed to `version_id`
+#' in `box_dl()` and `box_read()`.
+#' If the version of a file is one, then NULL is returned invisibly
 #' along with a helpful message.
 #' 
-#' The returned `data.frame` contains a variable, `file_version_id`, 
-#' which you can use with [box_dl()].
+#' * `box_current_version()` returns a `integer`, starting from 1, which can be passed
```

Comment:
how about `box_get_current_version()`? at some point also to rename (and soft-deprecate) `box_previous_versions()` to `box_file_get_info()` (I'd have to take a look at what gets returned)?

Reply:
I dropped this to `box_version(file_id)` in my responses. It feels succinct and fits with box_previous_versions()

Reply:
Sounds good! I wonder if this is a good opportunity to supersede, rather than to deprecate. 

Let me see if I can take a whack at it - here's an example of the idea: https://github.com/tidyverse/dplyr/blob/master/R/sample.R